homework-submission/myapp/src/Category.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Items from './Items';
+
+class Category extends React.Component {
+
+    todo = Items.map((todoItems, index) => (
+        <li key={index}>
+            {`${todoItems.description} , ${todoItems.deadLine}`}
+        </li>
+    ));
+
+    render() {
+        return (
+            <ul className='items'>
```

Comment:
Create another component `<Item description="" deadline="" />` which uses the `<li></li>`, instead of looping over `<li>`, use the component that you created.